packages/preview/scrutinize/0.2.0/typst.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "scrutinize"
+version = "0.2.0"
+entrypoint = "src/lib.typ"
+authors = [
+  "Clemens Koza <https://github.com/SillyFreak/>"
+]
+license = "MIT"
+description = "A library for building exams, tests, etc. with Typst"
+repository = "https://github.com/SillyFreak/typst-packages/tree/main/scrutinize"
+categories = ["model", "scripting", "office"]
+disciplines = ["education"]
+keywords = ["exam", "test", "school", "teaching"]
```

Comment:
@laurmaedje  I chose both model and scripting because the grading aspect of the library adds semantic information to the document, and then provides scripting tools to process that information. Also "office" because for a teacher, exams are similarly "day-to-day" as an invoice.

Do these match your notions of the categories? I have no problem changing them again, I just thought it's more efficient to simply put what I thought best there, in case I got it right.

Reply:
Sounds good.